<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dobavit_proved</class>
 <widget class="QDialog" name="dobavit_proved">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Город</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Название соревнований</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Время</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Обновить IDсоревнований</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Добавить строку в таблицу</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Почта</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Статус соревнований (если соревнований прошли, то напишите победителя, если соревнований не проводились, то отметьте это)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
